src/component/contents/DraftEditorBlock.react.js

```diff
@@ -83,50 +77,49 @@ class DraftEditorBlock extends React.Component {
    * will miss out on the browser natively scrolling to that position.
    *
    * To replicate native behavior, if the block overlaps the selection state
-   * on mount, force the scroll position. Check the scroll state of the scroll
-   * parent, and adjust it to align the entire block to the bottom of the
-   * scroll parent.
+   * on mount, force the scroll position. Check the visible bounds of the scroll
+   * parent against the bounds of the new block and scroll up or down to bring
+   * it into view.
    */
   componentDidMount(): void {
-    var selection = this.props.selection;
-    var endKey = selection.getEndKey();
+    const selection = this.props.selection;
+    const endKey = selection.getEndKey();
+
     if (!selection.getHasFocus() || endKey !== this.props.block.getKey()) {
       return;
     }
 
-    var blockNode = ReactDOM.findDOMNode(this);
-    var scrollParent = Style.getScrollParent(blockNode);
-    var scrollPosition = getScrollPosition(scrollParent);
-    var scrollDelta;
+    const blockElement = this._element;
+    if (blockElement == null) {
```

Comment:
⭐️ ⭐️ Why use a double equals here when we could just do a falsy check? `if (!blockElement) { return; }`

Reply:
I just updated that, it came from Draft's implementation.